### PR TITLE
Create layer_clear_selective.lua

### PR DESCRIPTION
**Pull request by cv-on-hub**
new script - clear chosen layer of current selection

### CODE REVIEW

File: src/layer_clear_selective.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    finaleplugin.Notes = [[
+        Clear all music from the chosen layer in the surrently selected region. 
+        (Note that all of a measure's layer will be cleared even if it is partially selected).
+    ]]
+    return "Clear layer selective", "Clear layer selective", "Clear the chosen layer"
+end
+
+-- RetainLuaState will return global variable: clear_layer_number
+
+function clear_chosen_layer(layer_number)
```
Comment (Nick-Mazuk) [inline src/layer_clear_selective.lua:18]: There's a library function `layers.clear`. Would that work here?

https://www.finalelua.com/docs/library/layer#clear

Comment (cv-on-hub) [reply src/layer_clear_selective.lua]: Yes of course. Silly me. Amended version follows. (This is a really old script that I've dragged up to fin-lua expectations).

**Comment by cv-on-hub** [comment]
Is that last `Merge branch` the right thing to do? I just wanted my cv-on-hub fork to match the finale-lua "dist" contents.

**Review by Nick-Mazuk — APPROVED** [review]
LGTM

**Comment by Nick-Mazuk** [comment]
> Is that last `Merge branch` the right thing to do? I just wanted my cv-on-hub fork to match the finale-lua "dist" contents.

It depends. That's what's called a merge commit (in case you want to Google it).

Some people say to never do it since it makes a messy commit history. Personally, I don't care since almost no one ever looks at the commit history, so I'm going to go ahead and approve and merge.

**Pull request merged**